As for shiboken2 package, makes file naming consistent across architectures for
so libs and cmake files: removes intermediate suffix like ".cpython-36m-x86_64-linux-gnu".
Avoid overwriting the "real" cmake file with the generic wrapper's one (same
name in such a case)

--- a/sources/pyside2/libpyside/CMakeLists.txt
+++ b/sources/pyside2/libpyside/CMakeLists.txt
@@ -177,8 +177,6 @@
      PATH_VARS PYSIDE_PYTHONPATH PYSIDE_TYPESYSTEMS PYSIDE_GLUE
 )
 
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/PySide2Config.cmake.in"
-               "${CMAKE_CURRENT_BINARY_DIR}/PySide2Config.cmake" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/PySide2ConfigVersion.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/PySide2ConfigVersion.cmake" @ONLY)
 
@@ -195,9 +193,6 @@
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/pyside2${pyside2_SUFFIX}.pc"
         DESTINATION "${LIB_INSTALL_DIR}/pkgconfig")
 
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/PySide2Config.cmake"
-        DESTINATION "${LIB_INSTALL_DIR}/cmake/PySide2-${BINDING_API_VERSION}")
-
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/install/PySide2Config${SHIBOKEN_PYTHON_CONFIG_SUFFIX}.cmake"
         DESTINATION "${LIB_INSTALL_DIR}/cmake/PySide2-${BINDING_API_VERSION}")
 
